File: Server/SuperBoard/src/main/java/com/narara/superboard/boardmember/interfaces/dto/MemberResponseDto.java
package com.narara.superboard.boardmember.interfaces.dto;

import lombok.Builder;

@Builder
public record MemberResponseDto(
        Long memberId,
        String memberEmail,
        String memberNickname,
        String memberProfileImgUrl,
        String authority,
        Boolean isDeleted
) {
}

File: Android/core/model/src/main/java/com/ssafy/model/board/MemberResponseDTO.kt
package com.ssafy.model.board

import com.ssafy.model.member.Authority
import com.ssafy.model.with.DataStatus

data class MemberResponseDTO(
    val memberId: Long,
    val authority: Authority,
    val isRepresentative: Boolean,
    val memberEmail: String,
    val memberNickname: String,
    val memberProfileImgUrl: String?,

    @Transient
    val componentId: Long = 0L,
    @Transient
    val isStatus: DataStatus = DataStatus.STAY
)
